### PR TITLE
Use `nextAction` from SDK

### PR DESCRIPTION
**Pull request by D4nte**
Last PR for the Recommend Unique Action project.

### CODE REVIEW

File: api_tests/tests/erc20_halight.ts [hunk 1]
```diff
@@ -16,15 +16,15 @@ it(
         await alice.createHerc20HalightSwap(bodies.alice);
         await bob.createHerc20HalightSwap(bodies.bob);
 
-        await alice.init();
+        await expect(alice.next()).resolves.toBe("init");
```
Comment (thomaseizinger) [inline api_tests/tests/erc20_halight.ts:19]: Could we reduce the noise in writing tests here by introducing something like `alice.assertAndExecuteNext("init")`.

Naming of the function subject to be improved :)

Comment (D4nte) [reply api_tests/tests/erc20_halight.ts:19]: Yes, just wanted to check first that my peers agree on the assertion of the step.

Comment (thomaseizinger) [reply api_tests/tests/erc20_halight.ts:19]: Oh definitely, I actually wanted to make the above a question but forgot the question mark, lol :D

**Comment by D4nte** [comment]
So Actually I shot myself in the foot because I cannot upgrade to the latest SDK as the latest SDK contain hbit changes that were not yet integrated in the e2e tests. I believe @thomaseizinger told me so...

**Pull request merged**